scripts/lib/CIME/buildlib.py

```diff
@@ -82,4 +82,5 @@ def run_gmake(case, compclass, libroot, bldroot, libname="", user_cppdefs=""):
     if user_cppdefs:
         cmd = cmd + "USER_CPPDEFS='{}'".format(user_cppdefs )
 
-    run_cmd_no_fail(cmd, combine_output=True)
+    _, out, _ = run_cmd(cmd, combine_output=True)
```

Comment:
buildlib is called by run_cmd with stdout redirected to a file and stderr combined with stdout.  Calling run_cmd again inside that environment with no stderr or stdout args causes output to get lost.